spec/lib/govuk_publishing_components/presenters/breadcrumb_selector_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "spec_helper"
+
+RSpec.describe GovukPublishingComponents::Presenters::BreadcrumbSelector do
+  subject do
+    described_class.new(
+      content_item,
+      request,
+      prioritise_taxon_breadcrumbs,
+      disable_ga4,
+    )
+  end
+  let(:example) { %w[guide guide] }
+  let(:content_item) { example_document_for(example.first, example.second) }
+  let(:path) { content_item["base_path"] }
+  let(:query_parameters) { {} }
+  let(:request) { instance_double("ActionDispatch::Request", path:, query_parameters:) }
+  let(:prioritise_taxon_breadcrumbs) { false }
+  let(:disable_ga4) { false }
+
+  def example_document_for(schema_name, example_name)
+    GovukSchemas::Example.find(schema_name, example_name:)
+  end
+
+  describe "#initialize" do
+    it "does not raise exception" do
+      expect { subject }.not_to raise_error
+    end
+  end
+
+  describe "#breadcrumbs" do
+    context "when content item is corporate information page with parent" do
+      let(:example) { %w[corporate_information_page corporate_information_page_complaints] }
+
+      it "returns breadcrumbs based on ancestors" do
```

Comment:
You're saying based on ancestors here, but refer to them as parent breadcrumbs in the other test. Could they both be the same pls

Reply:
Fixed.